discourse.Dockerfile

```diff
@@ -1,24 +1,14 @@
 FROM ubuntu:focal
 
-# ARGS become environment variables, but can be overridden using the
-# --build-arg var=foo option to docker build. This allows you to have a
-# default build image, but customize certain options such as app version or
-# userids, etc.
-ARG CONTAINER_APP_VERSION
-ARG CONTAINER_APP_USERNAME
-ARG CONTAINER_APP_UID
-ARG CONTAINER_APP_GROUP
-ARG CONTAINER_APP_GID
-
 # Used in Launchpad OCI Recipe build to tag the image.
-LABEL org.label-schema.version=${CONTAINER_APP_VERSION:-v2.7.10}
+LABEL org.label-schema.version=v2.7.10
 
 # Copy any args we got into the environment.
-ENV CONTAINER_APP_VERSION ${CONTAINER_APP_VERSION:-v2.7.10}
-ENV CONTAINER_APP_USERNAME ${CONTAINER_APP_USERNAME:-discourse}
-ENV CONTAINER_APP_UID ${CONTAINER_APP_UID:-200}
-ENV CONTAINER_APP_GROUP ${CONTAINER_APP_GROUP:-discourse}
-ENV CONTAINER_APP_GID ${CONTAINER_APP_GID:-200}
+ENV CONTAINER_APP_VERSION v2.7.10
+ENV CONTAINER_APP_USERNAME discourse
+ENV CONTAINER_APP_UID 200
+ENV CONTAINER_APP_GROUP discourse
+ENV CONTAINER_APP_GID 200
```

Comment:
Why do you remove that in a PR which is not related to that at all ?
It was probably added for a purpose, I think by Niels. Did you discuss that with him ?

Globally speaking I see too many PRs which are changing things out of the scope of the ticket, leading to some changes merged under the wood.